ecf/dev/del.sh

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Does this script need to live in a public repository?
It is hardwired for a single user.

Reply:
Good point. I will keep this in my branch for development test and remove it from "develop" branch

Reply:
The script is deleted